client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -106,7 +82,12 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 					</div>
 					<div className="steps__summary-column">
 						<h3 className="steps__summary-heading">{ translate( 'Continue your site setup:' ) }</h3>
-						{ this.renderTodo() }
+						<TodoLinksComponent
+							siteId={ siteId }
+							siteSlug={ siteSlug }
+							siteUrl={ siteUrl }
+							translate={ translate }
```

Comment:
Do we benefit anything if we pass this one down, vs. calling `localize` in the `TodoLinks`?